webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala

```diff
@@ -286,8 +286,33 @@ class TestNodeAndGlobalParameterLookup extends Specification {
       test(all(_), """${something.cfengine}""", List(NonRudderVar("something.cfengine")))
     }
 
-    "parse a non valid cfengine variable as a string" in {
-      test(all(_), """${something . cfengine}""", List(CharSeq("${something . cfengine"), CharSeq("}")))
+    "parse a non rudder param variable with accessor" in {
+      test(all(_), """${something.cfengine[foo]}""", List(NonRudderVar("something.cfengine", List("foo"))))
```

Comment:
Space/return are forbidden know ? I *think* some user had asked to them to be able to put long var on several lines, but I'm not sure and don't find it back. So I would say, not too grave (perhaps even a net win), but will need a note in release note.

Reply:
They are parsed as `UnsafeRudderVar` [here](https://github.com/clarktsiory/rudder/blob/bug_25111/_/webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala#L315), which are supported on Linux (old behavior is kept), and not supported in Windows !